test/utils/unidriver/ReactBase.js

```diff
@@ -61,6 +61,8 @@ export function ReactBase(base) {
       Simulate.mouseEnter(await htmlElement(), eventData),
     mouseLeave: async eventData =>
       Simulate.mouseLeave(await htmlElement(), eventData),
+    enterText: async eventData =>
```

Comment:
I see the convention is to use the same method name, so maybe:
```suggestion
    beforeInput: async eventData =>
```